en/docs/learn/authentication-session-persistence.md

```diff
@@ -107,10 +107,10 @@ few tasks to clean the data in the database for some given conditions.
 
 #### Task Pool
 
-We are storing above data within a critical path which controls the
-authentication. So we have introduced a pooling mechanism to put the
+We employed task pool on some older releases, when storing above data within a critical path which controls the
```

Comment:
shall we try to avoid using "we" and rephrase this to something like "On older releases of WSO2 Identity Server, the task pool was employed when storing...." ?